src/DynamoCoreWpf/Views/PackageManager/PackagePathView.xaml.cs

```diff
@@ -14,7 +15,7 @@ namespace Dynamo.Wpf.Views.PackageManager
     /// <summary>
     /// Interaction logic for PackagePathView.xaml
     /// </summary>
-    public partial class PackagePathView : Window
+    public partial class PackagePathView : System.Windows.Controls.UserControl
```

Comment:
could this be a breaking change ?

Reply:
yes, but like I mentioned the other day - we decided with Morpheus team that we're free to break WPF control classes.